webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedInitiatorDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedRequestDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedRequestMethods": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "91"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "requestDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "requestMethods": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "91"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
```

Comment:
`requestMethods` is not supported by Safari.

Reply:
```suggestion
                  "version_added": false
```